doc/topics/releases/3005.rst

```diff
@@ -1,7 +1,135 @@
 .. _release-3005:
 
 =============================================
-Salt 3005 Release Notes - Codename Phosphorus
+Salt 3005 release notes - Codename Phosphorus
 =============================================
 
 Salt 3005 is currently under development.
+
+
+Python 3.5 and 3.6 Deprecation
+------------------------------
+
+This will be the last release we will support Python versions 3.5 and 3.6.
+In Salt release 3006, we will only support Python versions 3.7 and higher.
+Going forward, our policy will be to align with Python's supported versions.
```

Comment:
**Chore (future action, not blocking):** I don't yet have the official URL set up for the Salt Install Guide, but when we do, we should link to the policy where we state this. I've been waiting for Derek to return for putting that request in to Kirill in case Kirill wanted to ask Derek questions about the URL setup.